shared/src/validation-schemas/sign-up/sign-up.schema.ts

```diff
@@ -0,0 +1,15 @@
+import * as Joi from 'joi';
+import { UserCreatePayload } from '~/common/types/types';
+
+const SignupSchema = Joi.object<UserCreatePayload>({
+  firstName: Joi.string().min(3).max(15).required(),
```

Comment:
We have to make validation more strict for the nickname and password fields, but also make it more loose for the first and last name field (Maybe from 1 to 20)